### ISSUE TITLE
Identifier in the Add FX list

### ISSUE THREAD

**Issue by WolfInABowl** [opentoonz/opentoonz#3890]
**Is your feature request related to a problem? Please describe**

**Describe the solution you'd like**

**Comment by DarrenTAnims** [comment]
Sounds sensible to me.

**Comment by artisteacher** [comment]
Some sort of indicator would be helpful (also for Insert FX and Replace FX) but could get very busy visually. If OT had Tahoma's status bar, that would be a great place to give more information about the FX without adding visual clutter. 

Personally, the first thing I would do to improve adding/inserting/replacing FX would be adding a search field.

**Comment by gab3d** [comment]
there already is a search field in FX Browser, or you mean something different?

**Comment by RodneyBaker** [comment]
Just to clarify, the search field is available in one location related to the FX Browser... via New FX in xsheet/timeline... as it was only added to that process.  When accessing via other means it won't appear.  Here we see the search field via New FX:

![image](https://user-images.githubusercontent.com/2576432/114469074-17295200-9bb2-11eb-8c83-6240a9cdf23d.png)

And here we don't directly via schematic using Add FX:

![image](https://user-images.githubusercontent.com/2576432/114469378-a20a4c80-9bb2-11eb-8ead-f21429fe17f8.png)

**Comment by artisteacher** [comment]
Yeah, I always forget about the FX browser. I always use the context menu options.
